backend/headless/backend.c

```diff
@@ -84,6 +87,44 @@ static void handle_display_destroy(struct wl_listener *listener, void *data) {
 	backend_destroy(&backend->backend);
 }
 
+static bool check_rgba8_ext(void) {
+	const char *exts_str = (const char *)glGetString(GL_EXTENSIONS);
+	if (exts_str == NULL) {
+		wlr_log(WLR_ERROR, "glGetString(GL_EXTENSIONS) failed");
+		return false;
+	}
+	return strstr(exts_str, "GL_OES_rgb8_rgba8") != NULL;
```

Comment:
Is... is this really how we're checking for GL extensions now?

Reply:
It looks like ```RGBA8_OES``` is mandatory when ```GL_OES_required_internalformat"``` is supported. ```OES_rgb8_rgba8``` only limits the availability of ```RGB8```.

https://www.khronos.org/registry/OpenGL/extensions/OES/OES_required_internalformat.txt

Reply:
>Is... is this really how we're checking for GL extensions now?

I guess I'll add a `wlr_gles2` helper function for this.

>It looks like RGBA8_OES is mandatory when GL_OES_required_internalformat" is supported. OES_rgb8_rgba8 only limits the availability of RGB8.

I don't get it. We only care about RGBA8_OES, which is defined in OES_rgb8_rgba8.

Reply:
> I don't get it. We only care about RGBA8_OES, which is defined in OES_rgb8_rgba8.

OES_required_internalformat defines ```RGBA8_OES``` as well, so it could be usable even if OES_rgb8_rgba8 is not supported.

Reply:
Good point. Added all of these.